lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -54,7 +55,7 @@ class UnusedBytes extends Audit {
    * @return {string}
    */
   static bytesToMsString(bytes, networkThroughput) {
-    return (Math.round(bytes / networkThroughput * 100) * 10).toLocaleString() + 'ms';
+    return Util.formatMilliseconds( Math.round(bytes / networkThroughput * 100) * 10, 1);
```

Comment:
I think you could reduce this to just `formatMilliseconds(bytes / networkThroughout * 1000)`